### ISSUE TITLE
Disabling formatting when replying removes any indication what the original message is

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/mail#7414]
### Steps to reproduce

1. Reply to an email
2. Disable formatting from the menu

Originally looks like this:
![image](https://user-images.githubusercontent.com/925062/195298664-daff0ec1-9b8d-4c9f-bbe4-6e8b7c7e6d58.png)

### Expected behavior

Quoted text / original message is prepended with `> ` as is standard:

![image](https://user-images.githubusercontent.com/925062/195299174-5816ec0b-f6c8-4f87-b610-27f41f4716c8.png)

### Actual behavior

Quoted text / original message is indistinguishable from normal text

![image](https://user-images.githubusercontent.com/925062/195298730-f442d63b-8cc2-49c3-9f97-8813680ae400.png)

### Mail app version

Current on cloud.nc.com

**Comment by ChristophWurst** [comment]
Label dropped because it was a duplicate of the attribute at https://github.com/orgs/nextcloud/projects/61. The priority does **not** change.

**Comment by kesselb** [comment]
Hi, good catch! 

The main difference between "Plain text" and "Rich text" are the loaded plugins for CKEditor. 
Internal state/data is always HTML until the message is sent.

There is one exception: Quotes :ghost: 

https://github.com/nextcloud/mail/blob/86edd50d667c0f1fb2fa72e0acd8652885ea4197/src/ReplyBuilder.js#L37-L38

- Plain text: Prepend every line, from the original message, with `>`
- Rich text: Wrap the original message with `<blockquote></blockquote>` 

Convert from rich to plain:

- Look for `<blockquote>` element inside `<quote>`
- Prepend every text node with `>`
- Unwrap text nodes

Convert from plain to rich:

- Look for lines starting with `>` inside `<quote>`
- Remove `>` and move text node to `<blockquote>` element


Pain points: 

- Convert from plain to rich: Quote inside Quote ;) 
- Is it possible to return a promise from a CKEditor plugin? The conversion should be done before changing the actual editor mode (because an editor mode change will destroy the editor instance)

Things to discuss:

![image](https://user-images.githubusercontent.com/3902676/197802024-829f0ace-a418-47e3-86f8-9c59a96a1418.png)

Set the writing mode to plain text. In my tests with Mail 2.0 the editor mode depends on the quoted message. 
HTML message = rich, text message = plain. 

@ChristophWurst @jancborchardt Is this the expected behavior or a bug?

**Comment by ChristophWurst** [comment]
What could we do instead? Always keep the blockquote, even for plain text messages and only convert them before sending?
